mybatis-spring-boot-autoconfigure/src/main/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfiguration.java

```diff
@@ -206,7 +208,9 @@ public void setResourceLoader(ResourceLoader resourceLoader) {
 
     @PostConstruct
     public void afterPropertiesSet() {
-      log.debug(String.format("No %s found.", MapperFactoryBean.class.getName()));
+      if (log.isDebugEnabled()) {
+        log.debug(String.format("No %s found.", MapperFactoryBean.class.getName()));
+      }
```

Comment:
Hi @hazendaz , thanks for your comment.

In this case, your understanding and opinion is correct. 
However, i think general coding style to use the `isDebugEnable` method when debug message is dynamically.

What do you think ?